### PR TITLE
Fix/selection bar

### PR DESCRIPTION
**Pull request by Crash--**
There is a lot to be done for the SelectionBar component, but let's do it gradually.

First: SelectionBar action can now have an icon attribute to use instead of relying only on the Action name
Second: Added an example on the Styleguidist 

https://crash--.github.io/cozy-ui/react/#/Layout%20components?id=selectionbar

@ptbrowne it seems that the style for the SelectionBar is not exported / imported in the styleguidist. Do you know how I can add it?

### CODE REVIEW

**Comment by ptbrowne** [comment]
The styles were not applied to the component because the CSS rules were scoped under `role=application`. I put the rules under a mixin in the CSS implementation that I import under a class in the React style component. I also could not stop myself and did a tiny bit of refactoring, see the subsequent commits.

Other improvements that could be done>

- The docstring could be improved into a real docstring
- PropTypes could be improved with the shape of the actions
- Locales should be shipped with the component
- A deprecation warning could be logged when an action does not have actions[actionName].icon (could surely be done via the PropTypes)
- Semantic color variables could be used in the CSS
- I wonder why content-box is used, I find border-box to be easier to use and understand

File: react/SelectionBar/index.jsx [hunk 1]
```diff
@@ -23,13 +23,9 @@ actions = {
...
-const SelectionBar = ({
-  t,
-  actions,
-  selected,
-  hideSelectionBar,
-  breakpoints: { isMobile, isTablet }
-}) => {
+const SelectionBar = ({ actions, selected, hideSelectionBar }) => {
+  const { t } = useI18n()
+  const { isMobile, isTablet } = useBreakpoints()
```
Comment (Crash--) [inline react/SelectionBar/index.jsx:28]: Using useBreakpoins() is a BC if the APP doesn't use a `BreakpointsProvider`

**Comment by ptbrowne** [comment]
> Have you an idea of why we were doing that?

No, I guess it was a mistake.

File: react/SelectionBar/index.jsx (hunk 1 above)
Comment (ptbrowne) [reply react/SelectionBar/index.jsx:28]: Thanks, I've edited https://github.com/cozy/cozy-ui/pull/1607/commits/b97fd3cefabd616ed289eb25f3346fe09957b1dd to add a BREAKING CHANGE mention.

File: stylus/components/selectionbar.styl [hunk 2]
```diff
@@ -29,9 +29,6 @@ $selectionbar
         padding 0 rem(16)
         color   var(--white)
 
-    .SelectionBar-action-extra
-        display none
-
     .SelectionBar-action-close
```
Comment (JF-Cozy) [inline stylus/components/selectionbar.styl:32]: ce ne serait pas plutôt `SelectionBar-actionClose` suite aux changement du commit précédent ?

**Review by ptbrowne — APPROVED** [review]

**Pull request merged**